lib/mutant/ast/regexp.rb

```diff
@@ -0,0 +1,41 @@
+module Mutant
+  module AST
+    # Regexp source mapper
+    module Regexp
+      UNSUPPORTED_EXPRESSION_TYPE = :conditional
+      private_constant(*constants(false))
+
+      # Check if expression is supported by mapper
+      #
+      # @param expression [Regexp::Expression]
+      #
+      # @return [Boolean]
+      def self.supported?(expression)
+        return if expression.type.equal?(UNSUPPORTED_EXPRESSION_TYPE)
```

Comment:
Should this return `false` given that the documented return type is `Boolean`? It may mean adding a new spec, but for public interfaces I think this is worth the effort to have a clean, unambiguous interface.